app/src/main/java/com/concordia/mcga/fragments/TransportButtonFragment.java

```diff
@@ -59,41 +71,49 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         publicTransportFAB.setOnClickListener(this);
         shuttleFAB.setOnClickListener(this);
 
-        //Some test values
-        setWalkTime(1, 30);
-        setBikeTime(1, 0);
-        setCarTime(0, 30);
-        setPublicTransportTime(0, 45);
-        setShuttleTime(0, 30);
+        //Set initial transport type and icon
+        this.transportType = Transportation.PUBLIC_TRANSPORT;
+        swapIcons(Transportation.PUBLIC_TRANSPORT.getIconID());
 
         return view;
     }
 
     @Override
     public void onClick(View clickedView) {
+        toggle();
         switch (clickedView.getId()) {
```

Comment:
The default behavior is that nothing happens, also the buttons are meant to be static as to comply with our mockup, so the buttons aren't meant to be changed which is why there is no interface to add custom buttons.